development.md

```diff
@@ -0,0 +1,24 @@
+### Get  Started with Development 
+#
+
+## prerequisites
+
+- installed go if not installed. Follow the below link for more details
+
+[go installation] (https://go.dev/dl/)
```

Comment:
In markdown, I think you need to get rid of the space between the link name in square brackets and the link in parentheses.